From 801b7e6c5b230674c4491888b8516efaecdbcc18 Mon Sep 17 00:00:00 2001
From: Tony Lin <tony.lin@freescale.com>
Date: Mon, 22 Aug 2011 19:48:55 +0800
Subject: [PATCH 0358/2463] ENGR00139260-1 [ath6k wifi]remove drivers under
 drivers/net/wireless

instead we will use ath6k driver under drivers/staging

Signed-off-by: Tony Lin <tony.lin@freescale.com>
---
 drivers/net/wireless/Kconfig  |    6 +++++-
 drivers/net/wireless/Makefile |    6 +++++-
 2 files changed, 10 insertions(+), 2 deletions(-)

diff --git a/drivers/net/wireless/Kconfig b/drivers/net/wireless/Kconfig
index e8e1fbc..a8bc7cc 100644
--- a/drivers/net/wireless/Kconfig
+++ b/drivers/net/wireless/Kconfig
@@ -284,7 +284,11 @@ source "drivers/net/wireless/rtlwifi/Kconfig"
 source "drivers/net/wireless/wl1251/Kconfig"
 source "drivers/net/wireless/wl12xx/Kconfig"
 source "drivers/net/wireless/zd1211rw/Kconfig"
+<<<<<<< HEAD
 source "drivers/net/wireless/mwifiex/Kconfig"
-source "drivers/net/wireless/ath6kl/Kconfig"
+#source "drivers/net/wireless/ath6kl/Kconfig"
+=======
+#source "drivers/net/wireless/ath6kl/Kconfig"
+>>>>>>> ENGR00139260-1 [ath6k wifi]remove drivers under drivers/net/wireless
 
 endif # WLAN
diff --git a/drivers/net/wireless/Makefile b/drivers/net/wireless/Makefile
index a9ba769..d555677 100644
--- a/drivers/net/wireless/Makefile
+++ b/drivers/net/wireless/Makefile
@@ -57,5 +57,9 @@ obj-$(CONFIG_WL12XX_PLATFORM_DATA)	+= wl12xx/
 
 obj-$(CONFIG_IWM)	+= iwmc3200wifi/
 
+<<<<<<< HEAD
 obj-$(CONFIG_MWIFIEX)	+= mwifiex/
-obj-$(CONFIG_ATH6K_LEGACY)	+= ath6kl/
+#obj-$(CONFIG_ATH6K_LEGACY)	+= ath6kl/
+=======
+#obj-$(CONFIG_ATH6K_LEGACY)	+= ath6kl/
+>>>>>>> ENGR00139260-1 [ath6k wifi]remove drivers under drivers/net/wireless
-- 
1.7.7.4

